provisioner/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -70,3 +69,19 @@ ec2_info:
     architecture: x86_64
     filter: 'F5 Networks Prelicensed Hourly BIGIP-13.1*Good 25MBPS*'
     username: admin
+  splunk_enterprise:
+    owners: 309956199498
+    size: t2.micro
+    os_type: linux
+    disk_space: 10
+    architecture: x86_64
+    filter: 'RHEL-7.6_HVM-20190515-x86_64-0-Access2-GP2'
+    username: ec2-user
+  checkpoint_firewall:
+    owners: 309956199498
+    size: t2.micro
+    os_type: linux
+    disk_space: 10
+    architecture: x86_64
+    filter: 'RHEL-7.6_HVM-20190515-x86_64-0-Access2-GP2' #FIXME - needs to be something checkpoint
```

Comment:
For Checkpoint CloudGuard IaaS please change to `Check Point CloudGuard IaaS*PAYG-MGMT*R80.20*`